### ISSUE TITLE
Occasional crash on Pixel 5 Android 13

### ISSUE THREAD

**Issue by kyze8439690** [G00fY2/quickie#76]
Integrate with: `implementation("io.github.g00fy2.quickie:quickie-bundled:1.5.1")`

Crash log:
```
Fatal Exception: java.lang.IllegalStateException: MlKitContext has not been initialized
       at com.google.android.gms.common.internal.Preconditions.checkState(com.google.android.gms:play-services-basement@@18.1.0:2)
       at com.google.mlkit.common.sdkinternal.MlKitContext.getInstance(com.google.mlkit:common@@18.5.0:1)
       at com.google.mlkit.vision.barcode.BarcodeScanning.getClient(BarcodeScanning.java:3)
       at io.github.g00fy2.quickie.QRCodeAnalyzer$barcodeScanner$2.invoke(QRCodeAnalyzer.kt:24)
       at io.github.g00fy2.quickie.QRCodeAnalyzer$barcodeScanner$2.invoke(QRCodeAnalyzer.kt:18)
       at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
       at io.github.g00fy2.quickie.QRCodeAnalyzer.getBarcodeScanner(QRCodeAnalyzer.kt:18)
       at io.github.g00fy2.quickie.QRCodeAnalyzer.analyze(QRCodeAnalyzer.kt:42)
       at androidx.camera.core.ImageAnalysis$$ExternalSyntheticLambda2.b(R8$$SyntheticClass:3)
       at androidx.camera.core.ImageAnalysisAbstractAnalyzer.lambda$analyzeImage$0(ImageAnalysisAbstractAnalyzer.java:285)
       at androidx.camera.core.ImageAnalysisAbstractAnalyzer$$InternalSyntheticLambda$8$864b24cc058c102b9ababe9f2eaf5f45916e7a7a0663a2b450741580c02ea23e$0.run(ImageAnalysisAbstractAnalyzer.java:66)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
       at java.lang.Thread.run(Thread.java:1012)
```

**Comment by G00fY2** [comment]
Hi and thanks for reaching out.

This library does nothing that could affect the default MlKitContext initialization. The initialization should happen automatically by the `com.google.mlkit.common.internal.MlKitInitProvider` content provider added by the MlKit dependency internally. There is nothing I can do to affect this.

Did you maybe disabled any content provider in your app/manifest?

Maybe you can check if the [ML Kit Vision Quickstart Sample App](https://github.com/googlesamples/mlkit/tree/master/android/vision-quickstart) is working for you.

There are also some issues in the [MlKit GitHub project](https://github.com/googlesamples/mlkit/issues?q=MlKitContext) which could may help you.

Otherwise please create a bug ticket in the [Google issue tracker](https://issuetracker.google.com/issues?q=status:open%20componentid:897785&s=created_time:desc) and/or in their GitHub issues.

**Comment by kyze8439690** [comment]
Is it possible for quickie to catch this exception and invoke something like 'failure callback' instead?

**Comment by G00fY2** [comment]
Yeah should be able to catch this exception. Will try to add this in a future release. But thats nothing Google mentions/recommends in it's docs about MlKit. The issue should be fixed in MlKit. Maybe you can still create a ticket there.

**Comment by G00fY2** [comment]
This is fixed with f164aa9 (which is included in [1.5.2](https://github.com/G00fY2/quickie/releases/tag/1.5.2)).

**Comment by kyze8439690** [comment]
> Yeah we should be able to catch this exception. Will try to add this in a future release. But thats nothing Google mentions/recommends in it's docs about MlKit. The issue should be fixed in MlKit. Maybe you can still create a ticket there.

Thanks for that, I see somebody have open issue on mlkit github repo issue already.

https://github.com/googlesamples/mlkit/issues/565
